interface/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "solana-stake-interface"
+version = "0.1.0"
+description = "Instructions and constructors for the Stake program"
+repository = "https://github.com/solana-program/stake"
+edition = "2021"
+readme = "README.md"
+license-file = "../LICENSE"
+
+[dependencies]
+borsh = { version = "1.5.1", features = ["derive", "unstable__schema"], optional=true }
+borsh0-10 = { package = "borsh", version = "0.10.3", optional=true }
+num-traits = "0.2"
+serde = { version = "1.0.210", features = ["derive"] }
+serde_derive = "1.0.210"
```

Comment:
Is it possible to make these optional with a `serde` crate feature, as we've been doing? And then we can enable the `bincode` / `serde` features on dependencies optionally. It'll be a little messy becauase all of the instruction creators require bincode / serde, but this way, in the future we can move the instruction creators to use something else.

Reply:
How would that work? Do we make the whole instruction creator optional as well under a different feature (e.g., `"bincode"`)? So enabling `"bincode"` would enable `"serde"`, since `Instruction::new_with_bincode` needs both.

Reply:
Yes exactly